<!DOCTYPE html>
<html lang="en">
  <head>
	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-172431779-1"></script>
	<script>
  		window.dataLayer = window.dataLayer || [];
  		function gtag(){dataLayer.push(arguments);}
  		gtag('js', new Date());

  		gtag('config', 'UA-172431779-1');
	</script>

    <meta charset="UTF-8">
    <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
    <meta name="description" content="Wei Vivian Li's Homepage">
    <title>Wei Vivian Li Lab</title>
    <link rel='stylesheet' href='./bootstrap/css/style.css' type='text/css'>
    <!--[if lte IE 8]>
    <link rel='stylesheet' href='./css/style-ie.css' type='text/css'>
    <![endif]-->
    <script src="./bootstrap/js/jquery-3.3.1.min.js"></script>
  </head>

  <body>
    <div id="top-banner" class="banner">
      <!-- <img style="position:absolute;left:450px;" src="./img/title-logo.png"> -->
    </div>
    <div id="main-body" class="wrapper">
      <div id="site-title">
        <div id="name">Wei Vivian Li
          <div id="subname">PhD in Statistics</div>
        </div>
        <div id="access">
          <ul id="nav">
            <li><a id="link-contact" href="#contact" onclick="changeNavState(id)">TEACHING</a></li>
            <li><a id="link-projects" href="#projects" onclick="changeNavState(id)">PUBLICATION</a></li>
            <li><a id="link-research" href="#research" onclick="changeNavState(id)">RESEARCH</a></li>
            <li><a id="link-about" href="#about" onclick="changeNavState(id)">ABOUT</a></li>
            <li><a id="link-home" href="#home" onclick="changeNavState(id)">HOME</a></li>
          </ul>
        </div>
      </div>
      <div id="content">
        <!-- will be filled by javascript -->
      </div>
    </div>
    <!-- home tab -->
    <script id="home-content" type="text/x-content">
      <div id="home-gallery">
        <div id="gallery-background" class="gallery-content">
          <span class="gallery-photo" style="opacity:1;filter:Alpha(opacity=100)">
            <img src="./bootstrap/img/gallery-photo1.jpg">
          </span>        
          <span class="gallery-photo" style="opacity:0;filter:Alpha(opacity=0)">
            <img src="./bootstrap/img/gallery-photo2.jpg">
          </span>
          <span class="gallery-photo" style="opacity:0;filter:Alpha(opacity=0)">
            <img src="./bootstrap/img/gallery-photo3.jpg">
          </span>
          <!-- two arrows switching photos -->
          <span id="arrows-left" onclick="window.clearInterval(timer); changePhotoLeft();timer = window.setInterval(changePhotoLeft,4000);">
            <img src="./bootstrap/img/arrows-left.png">
          </span>
          <span id="arrows-right" onclick="window.clearInterval(timer); changePhotoRight();timer = window.setInterval(changePhotoRight,4000);">
            <img src="./bootstrap/img/arrows-right.png">
          </span>
        </div>
        <div id="gallery-text" class="gallery-content">
          <h1>Welcome to Vivian Li Lab</h1>
          <div id="gallery-description">
            <p id="gallery-description-showing">A collection of my research and teaching activities</p>
            <p id="gallery-description-hidden"></p>
          </div>
        </div>
      </div>
      <div>
        <img src="./bootstrap/img/gallery-bottom.png" style="width:940px;height:40px;margin-bottom:24px;display:none">
      </div>

      <div class="container-outside">

        <div class="container-block">
          <h1>News</h1>
          <div class="container-description">
            <ul>
              <li><b>08/2021</b> Our R35 Maximizing Investigators' Research Award proposal was funded by the National Institute of General Medical Sciences! The grant will support our work on multi-scale analysis of single-cell transcriptomes. </li>
              <li><b>07/2021</b> New Paper! Our manuscript, Selecting gene features for unsupervised analysis of single-cell gene expression data, was accepted by Briefings in Bioinformatics! Congratulations to Jie!</li>
              <li><b>07/2021</b> Our R21 grant proposal in collaboration with Dr. Zhiping Pang at Rutgers was funded by the National Institute of Mental Health!</li>
              <li><b>06/2021</b> New paper! Our manuscript in collaboration with Dr. Tian at the Wistar Institute, MAAPER: model-based analysis of alternative polyadenylation using 3'end-linked reads, was accepted by Genome Biology!</li>
              <li><b>05/2021</b> A visiting student in the lab, Shiwei Fu, has received the Rutgers Presidential Fellowship and will join our lab as a PhD student this fall. Congratulations to Shiwei!</li>
            </ul>
            <details>
              <summary>Click to see all.</summary>
            <ul>
              <li><b>03/2021</b> New preprint! Our manuscript about analyzing alternative polyadenylation using using 3’ end-linked reads, is now available on <a href="https://www.biorxiv.org/content/10.1101/2021.03.21.436343v1" target="_blank">bioRxiv</a>.</li>
              <li><b>01/2021</b> New paper! Our manuscript, scLink: inferring sparse gene co-expression networks from single-cell expression data, was accepted by Genomics, Proteomics & Bioinformatics. Congratulations to Yanzeng!</li>
              <li><b>09/2020</b> New preprint! Our manuscript about constructing gene co-expression networks in single cells is now available on <a href="https://www.biorxiv.org/content/10.1101/2020.09.19.304956v1" target="_blank">bioRxiv</a>.</li>
              <li><b>09/2020</b> Vivian received the Busch Biomedical Grant Award.</li>
              <li><b>06/2020</b> Vivian received the BERD Method Grant from New Jersey Alliance for Clinical and Translational Science (<a href="https://njacts.rbhs.rutgers.edu/" target="_blank">NJ ACTS</a>).</li>
            </ul>
            </details>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div id="quick-links" class="container-block">
          <h1>Quick Links</h1>
          <div class="container-description">
            <div class="container-desc-col-4">
              <a href="https://github.com/Vivianstats" target="_blank">
                <span><img src="./bootstrap/img/my-github-logo.png"></span>
              </a>
            </div>

            <div class="container-desc-col-4">
              <a href="https://twitter.com/vivianstats" target="_blank">
                <span><img src="./bootstrap/img/my-twitter-logo.png"></span>
              </a>
            </div>

            <div class="container-desc-col-4">
              <a href="https://scholar.google.com/citations?user=HzjANkQAAAAJ" target="_blank">
                <span><img src="./bootstrap/img/my-google-scholar-logo.png"></span>
              </a>
            </div>

            <div class="container-desc-col-4">
              <a href="https://www.researchgate.net/profile/Wei_Vivian_Li" target="_blank">
                <span><img src="./bootstrap/img/my-research-gate-logo.png"></span>
              </a>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->
      </div> <!-- container-outside -->
    </script>

    <!-- research tab -->
    <script id="research-content" type="text/x-content">
      <div class="container-outside">

	      <div class="container-block">
          <h1>Single-cell Genomics</h1>
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-maaper.png">
            </span>
            <div id="research-text" class="container-text">
              <div>
                <h3>Analyzing Alternative Polyadenylation (APA)</h3>
                <p>MAAPER is a computational method for examining APA using 3' end-linked reads. MAAPER uses a probabilistic model to predict PASs for 3' end-linked  reads with high accuracy and sensitivity, and examines different types of APA events, including those in 3'UTRs and introns, with robust statistics. We show MAAPER's accuracy with data from both bulk and single cell RNA samples and its applicability in unpaired or paired experimental designs. [ <a href="https://www.biorxiv.org/content/10.1101/2021.03.21.436343v1" target="_blank">Preprint</a> ]</p>
              </div>
            </div> <!-- container-text -->
          </div> <!-- container-description -->
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-sclink.png">
            </span>
            <div id="research-text" class="container-text">
              <div>
                <h3>Constructing Gene Co-expression Networks</h3>
                <p>A system-level understanding of the regulation and coordination mechanisms of gene expression is essential to studying the complexity of biological processes in health and disease. Therefore, we propose the scLink method, which uses statistical network modeling to understand the co-expression relationships among genes and construct sparse gene co-expression networks from single-cell gene expression data. [ <a href="https://doi.org/10.1016/j.gpb.2020.11.006" target="_blank">Paper</a> ]</p>
              </div>
            </div> <!-- container-text -->
          </div> <!-- container-description -->
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-scimpute.png">
            </span>
            <div id="research-text" class="container-text">
              <div>
                <h3>Imputation of scRNA-seq Data</h3>
                <p>scImpute is a statistical method to accurately and robustly impute the dropouts in scRNA-seq data. scImpute automatically identifies likely dropouts, and only perform imputation on these values without introducing new biases to the rest data. scImpute is shown to identify likely dropouts, enhance the clustering of cell subpopulations, improve the accuracy of differential expression analysis, and aid the study of gene expression dynamics. [ <a href="https://www.nature.com/articles/s41467-018-03405-7" target="_blank">Paper</a> ]</p>
              </div>
            </div> <!-- container-text -->
          </div> <!-- container-description -->
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-scdesign.png">
            </span>     
            <div id="research-text" class="container-text">
              <div>
                <h3>Rational scRNA-seq Experimental Design</h3>
				  <p>We present a flexible and robust simulator, scDesign, the first statistical framework for researchers to quantitatively assess practical scRNA-seq experimental design in the context of differential gene expression analysis. In addition to experimental design, scDesign also assists computational method development by generating high-quality synthetic scRNA-seq datasets under customized experimental settings. 
				  <p>[ <a href="https://doi.org/10.1093/bioinformatics/btz321" target="_blank">scDesign</a> ] [ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02367-2" target="_blank">scDesign2</a> ]</p>
				  </p>
              </div>        
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>RNA Transcript Assembly and Quantification</h1>
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-aide.png">
            </span>	    
            <div id="research-text" class="container-text">
              <div>
			    <h3>Annotation-assisted Transcript Reconstruction</h3>
				<p>AIDE is a statistical method which identifies full-length mRNA isoforms from a novel perspective: using the likelihood ratio test to find novel isoforms in a stepwise manner given annotated isoforms, by prioritizing and selectively borrowing information from the annotated isoforms. AIDE is the first approach that directly controls false isoform discoveries by implementing the statistical model selection principle. [ <a href="https://genome.cshlp.org/content/29/12/2056" target="_blank">Paper</a> ]</p>
              </div>	      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-msiq.png">
            </span>	    
            <div id="research-text" class="container-text">
              <div>
			    <h3>Transcript Quantification of Multiple RNA-seq Samples</h3>
				<p>We develop a method, which we call "joint modeling of multiple RNA-seq samples for accurate isoform quantification" (MSIQ), for more robust isoform quantification by integrating multiple RNA-seq samples under a Bayesian framework. Our method aims to (1) identify a consistent group of samples with homogeneous quality and (2) improve isoform quantification accuracy by jointly modeling multiple RNA-seq samples and allowing for higher weights on the consistent group. [ <a href="https://projecteuclid.org/euclid.aoas/1520564482" target="_blank">Paper</a> ]</p>
              </div>	      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Comparative Bioinformatics</h1>
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-trom.png">
            </span>	    
            <div id="research-text" class="container-text">
              <div>
			    <h3>Transcriptome Overlap Measure</h3>
				<p>TROM is a testing-based method for comparing transcriptomes within or between different species, and provide a different perspective about capturing transcriptomic similarity. The TROM method focuses on identifying associated genes that capture molecular characteristics of biological samples, and subsequently comparing the biological samples by testing the overlap of their associated genes. [ <a href="https://link.springer.com/article/10.1007/s12561-016-9163-y" target="_blank">Paper</a> ]</p>
              </div>	      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-epom.png">
            </span>	    
            <div id="research-text" class="container-text">
              <div>
			    <h3>Epigenome Overlap Measure</h3>
				<p>We study the relationships among tissue and cell types from a new perspective: the similarity of tissue and cell types in terms of histone modification marks evaluated in relevant chromatin states. The proposed EPOM measure is capable of finding a clear correspondence map of biological samples from different tissue and cell types. The identified associated enhancers and promoters provide a comprehensive catalog to study distinct biological processes and disease variants in different tissue and cell types. [ <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2303-9" target="_blank">Paper</a> ]</p>
              </div>	      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Statistical Learning</h1>
          <div class="container-description">
            <span id="research-photo">
              <img src="./bootstrap/img/research-npcs.png">
            </span>
            <div id="research-text" class="container-text">
              <div>
                <h3>Asymmetric Classification</h3>
                <p>We propose an umbrella algorithm to estimate the population type I error control achieved by multiple cost-sensitive learning approaches.  Our algorithm for the first time establishes a connection between cost-sensitive learning and the Neyman-Pearson classification paradigm, which minimizes the population type II error while enforcing an upper bound on the population type I error. This algorithm can be applied in many real problems, such as disease diagnosis. [ <a href="https://arxiv.org/abs/2012.14951" target="_blank">Paper</a> ]</p>
              </div>        
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

      </div> <!-- container-outside -->
    </script>

    <!-- projects tab -->
    <script id="projects-content" type="text/x-content">
      <div class="container-outside">
        <div class="container-block">
          <h1>Publication</h1>
          <div class="container-description">
            <p>Lab members are in <b>bold</b>.</p>
            <p>Corresponding authors are <uu>underlined</uu>.</p>

          	<h3>Preprints:</h3>
            <ul>
              <li><b>Li WV</b>, Tong X, and Li JJ. (2021). Bridging cost-sensitive and Neyman-Pearson paradigms for asymmetric binary classification. <b><i>arXiv</i></b>.
                <p>[ <a href="https://arxiv.org/abs/2012.14951" target="_blank">Link</a> ]</p></li>              
              <li>Roque W, Cuevas-Mora K, Sales D, <b>Li WV</b>, Rosas IO, and Romero F. (2020). Transcriptomics, metabolomics and lipidomics of chronically injured alveolar epithelial cells reveals similar features of IPF lung epithelium. <b><i>bioRxiv</i></b>.
                <p>[ <a href="https://www.biorxiv.org/content/10.1101/2020.05.08.084459v1" target="_blank">Link</a> ]</p></li>
            </ul>

          	<h3>Refereed Articles:</h3>
            <ul> 
              <li><b><uu>Li WV</uu></b>, Zheng D, Wang R, and <uu>Tian B</uu>. (2021). MAAPER: model-based analysis of alternative polyadenylation using 3'end-linked reads. <b><i>Genome Biology</i></b>, 22, 222.
                <p>[ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02429-5" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/MAAPER" target="_blank">Software</a> ]</p></li> 
              <li><b><uu>Li WV</uu></b> and <b>Li Y</b>. (2021). scLink: inferring sparse gene co-expression networks from single-cell expression data. <b><i>Genomics, Proteomics & Bioinformatics</i></b>, in press.
                <p>[ <a href="https://doi.org/10.1016/j.gpb.2020.11.006" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scLink" target="_blank">Software</a> ] [ <a href="https://rutgersbiostat.shinyapps.io/sclink/" target="_blank">Web App</a> ] [ <a href="https://www.recomb2021.org/accepted-papers" target="_blank">RECOMB 2021</a> ]</p></li>
              <li><b>Sheng J</b> and <b><uu>Li WV</uu></b>. (2021). Selecting gene features for unsupervised analysis of single-cell gene expression data. <b><i> Briefings in Bioinformatics</i></b>, bbab295.
                <p>[ <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab295/6341636?guestAccessKey=2f8cc725-e1bc-4ff1-905b-504931532ee4" target="_blank">Link</a> ]</p></li>
              <li>Zhang XS, Yin YS, Wang J, Battaglia T, Krautkramer K, <b>Li WV</b>, Li J, Brown M, Zhang M, Badri MH, Armstrong AJ, Strauch CM, Wang Z, Nemet I, Altomare N, Devlin JC, He L, Morton JT, Chalk JA, Needles K, Liao V, Mount J, Li H, Ruggles KV, Bonneau RA, Dominguez-Bello MG, Bäckhed F, Hazen SL, Blaser MJ. (2021). Maternal cecal microbiota transfer rescues early-life antibiotic-induced enhancement of type 1 diabetes in mice. <b><i>Cell Host & Microbe</i></b>, 29(8):1249-1265.
                <p>[ <a href="https://doi.org/10.1016/j.chom.2021.06.014" target="_blank">Link</a> ]</p></li>
              <li>Sun T, Song D, <b><uu>Li WV</uu></b>, and <uu>Li JJ</uu>. (2021).  scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured. <b><i>Genome Biology</i></b>, 22(1):1-37.
                <p>[ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02367-2" target="_blank">Link</a> ] [ <a href="https://www.recomb2021.org/accepted-papers" target="_blank">RECOMB 2021</a> ] </p></li>
              <li>Jiang R, <b>Li WV</b>, and Li JJ. (2021). mbImpute: an accurate and robust imputation method for microbiome data. <b><i>Genome Biology</i></b>, 22:192.
                <p>[ <a href="https://doi.org/10.1186/s13059-021-02400-4" target="_blank">Link</a> ]</p></li>
              <li>Zheng Y, Chen Z, Han L, Han Y, Zou X, Zhou B, Hu R, Hao J, Bai S, Xiao H, <b>Li WV</b>, Bueker A, Ma Y, Xie G, Yang J, Chen S, Li H, Cao J, and Shen L. (2020). Immune suppressive landscape in the human esophageal squamous cell carcinoma microenvironment. <b><i>Nature Communications</i></b>, 11(1):1-17.  
                <p>[ <a href="https://www.nature.com/articles/s41467-020-20019-0" target="_blank">Link</a> ]</p></li>  
              <li>Wang J, Deng F, Zeng F, Shanahan AJ, <b>Li WV</b>, and Zhang L. (2020). Predicting long-term multicategory cause of death in patients with prostate cancer: random forest versus multinomial model. <b><i>American Journal of Cancer Research</i></b>, 10(5):1344.
                <p>[ <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7269775/" target="_blank">Link</a> ]</p></li>
              <li>Xu J, Cheng Y, Yuan X, <b>Li WV</b>, and Zhang L. (2020). Trends and prediction in daily incidence of novel coronavirus infection in China, Hubei Province and Wuhan City: an application of Farr's law. <b><i>American Journal of Translational Research</i></b>, 12(4):1355.
                <p>[ <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7191162/" target="_blank">Link</a> ]</p></li>            	
              <li><b>Li WV</b>*, Li S*, Tong X, Deng L, Shi H, and Li JJ. (2019). AIDE: annotation-assisted isoform discovery and abundance estimation from RNA-seq data. <b><i>Genome Research</i></b>, 29:2056-2072.
                <p>[ <a href="https://genome.cshlp.org/content/29/12/2056" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/AIDE" target="_blank">Software</a> ] [ <a href="https://genome.cshlp.org/content/29/12.cover-expansion" target="_blank">Cover Story</a> ] [ <a href="https://www.aspph.org/rutgers-study-achieves-highest-precision-and-lowest-error-rates-in-mrna-discoveries-using-a-robust-bioinformatics-tool/" target="_blank">ASPPH News</a> ]</p></li>
              <li><b>Li WV</b> and Li JJ. (2019). A statistical simulator scDesign for rational scRNA-seq experimental design. <b><i>Bioinformatics</i></b>, 35(14):i41-i50.
                <p>[ <a href="https://academic.oup.com/bioinformatics/article/35/14/i41/5529133" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scDesign" target="_blank">Software</a> ]</p></li>
              <li>Ge X, Zhang H, Xie L, <b>Li WV</b>, Kwon SB, and Li JJ. (2019). EpiAlign: an alignment-based bioinformatic tool for comparing chromatin state sequences. <b><i>Nucleic Acids Research</i></b>, gkz287. 
                <p>[ <a href="https://doi.org/10.1093/nar/gkz287" target="_blank">Link</a> ] [ <a href="http://shiny.stat.ucla.edu:3838/EpiAlign/" target="_blank">Website</a> ] </p></li>
              <li><b>Li WV</b> and Li JJ. (2018). An accurate and robust imputation method scImpute for single-cell RNA-seq data. <b><i>Nature Communications</i></b>, 9:997.
	             <p>[ <a href="https://www.nature.com/articles/s41467-018-03405-7" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scImpute" target="_blank">Software</a> ] [ <a href="http://newsroom.ucla.edu/releases/ucla-scientists-merge-statistics-biology-to-produce-important-new-gene-computational-tool" target="_blank">UCLA News</a> ]</p></li>
              <li><b>Li WV</b>*, Zhao A, Zhang S, and Li JJ*. (2018). MSIQ: joint modeling of multiple RNA-seq samples for accurate isoform quantification. <b><i>Annals of Applied Statistics</i></b>, 12(1):510-539. 
	               <p>[ <a href="https://projecteuclid.org/euclid.aoas/1520564482" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/MSIQ" target="_blank">Software</a> ]</p></li>
	            <li><b>Li WV</b> and Li JJ. (2018). Modeling and analysis of RNA-seq data: a review from a statistical perspective. <b><i>Quantitative Biology</i></b>, 6(3):195-209. 
	              <p>[ <a href="https://link.springer.com/article/10.1007/s40484-018-0144-7" target="_blank">Link</a> ]</p></li>
              <li><b>Li WV</b>, Chen Y, and Li JJ. (2017). TROM: a testing-based method for finding transcriptomic similarity of biological samples. <b><i>Statistics in Biosciences</i></b>, 9(1):105-136.
	             <p>[ <a href="https://link.springer.com/article/10.1007/s12561-016-9163-y" target="_blank">Link</a> ] [ <a href="https://cran.r-project.org/web/packages/TROM/index.html" target="_blank">Software</a> ]</p></li>
              <li><b>Li WV</b>, Razaee ZS, and Li JJ. (2016). Epigenome overlap measure (EPOM) for comparing tissue/cell types based on chromatin states. <b><i>BMC Genomics</i></b>, 17(Supp 1):10. 
	             <p>[ <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2303-9" target="_blank">Link</a> ] [ <a href="https://github.com/ruochenj/EPOM_R" target="_blank">Software</a> ]</p></li>
            </ul>

            <h3>Dissertation:</h3>
            <ul>
              <li><b>Li, WV</b> (2019). Statistical methods for bulk and single-cell RNA sequencing data. PhD dissertation, <b><i>UCLA</i></b>.
	            <p>[ <a href="https://escholarship.org/uc/item/50z7659w" target="_blank">Link</a> ]</p></li>	             
            </ul>
          </div>
        </div>

        <div class="container-block">
          <h1>Software</h1>
          <div class="container-description-reverse" style="margin-top:60px;">
            <span>
              <img src="./bootstrap/img/github-logo.png">
            </span>
            <div>
              <h3>Single-cell Transcriptomics</h3>
              <ul>
	        <li><p><a href="https://github.com/Vivianstats/scImpute" target="_blank">sclmpute</a>: Accurate and robust imputation of scRNA-seq data</p></li>
	        <li><p><a href="https://github.com/Vivianstats/scDesign" target="_blank">scDesign</a>: A statistical simulator for rational single-cell experimental design</p></li>
          <li><p><a href="https://github.com/Vivianstats/scLink" target="_blank">scLink</a>: Inferring sparse gene co-expression networks in single cells</p></li>
          <li><p><a href="https://github.com/Vivianstats/MAAPER" target="_blank">MAAPER</a>: Analyzing alternative polyadenylation using 3' end-linked reads</p></li>
              </ul>
            </div>
          </div>

          <div class="container-description-reverse">
            <span>
              <img src="./bootstrap/img/github-logo.png">
            </span>
            <div>
	      <h3>Bulk Tissue Transcriptomics</h3>
	      <ul>
	        <li><p><a href="https://github.com/Vivianstats/AIDE" target="_blank">AIDE</a>: Annotation-assisted transcript reconstruction and quantification</p></li>
          <li><p><a href="https://github.com/Vivianstats/MSIQ" target="_blank">MSIQ</a>: Quantification of RNA transcripts from multiple RNA-seq samples</p></li>
          <li><p><a href="https://github.com/Vivianstats/MAAPER" target="_blank">MAAPER</a>: Analyzing alternative polyadenylation using 3' end-linked reads</p></li>          
	      </ul>
            </div>
          </div>

          <div class="container-description-reverse">
            <span>
              <img src="./bootstrap/img/cran-logo.png">
            </span>
            <div>
              <h3>Comparative Bioinformatics</h3>
              <ul>
                <li><p><a href="https://github.com/Vivianstats/TROM" target="_blank">TROM</a>: Comparing transcriptomes from the same or different species</p></li>
                <li><p><a href="https://github.com/ruochenj/EPOM_R" target="_blank">EPOM</a>: Comparing epigenomes of biological samples</p></li>
              </ul>
            </div>
          </div>
	  <!-- only to add a little space at bottom -->
	  <div style="height:40px;"></div> 
        </div>
      </div>
    </script>

    <!-- about tab -->
    <script id="about-content" type="text/x-content">
      <div class="container-outside">
        <div class="container-block">
          <h1>About PI</h1>
          <div class="container-description">
            <span id="about-me-photo">
              <img src="./bootstrap/img/about-me.png">
            </span>
            <div id="about-me-text" class="container-text">
              <div>
				<p>Vivian (Wei) Li is an Assistant Professor in the <a href="https://sph.rutgers.edu/concentrations/biostatistics-epidemiology/index.html" target="_blank">Department of Biostatistics and Epidemiology</a> at the <a href="https://www.rutgers.edu/" target="_blank">Rutgers University</a>. She is also a faculty member in <a href="https://molbiosci.rutgers.edu/current-students/phd-programs" target="_blank">Rutgers Graduate Programs in Molecular Biosciences</a> and <a href="https://iqb.rutgers.edu/" target="_blank">Graduate Programs in Quantitative Biomedicine</a>. Vivian obtained her Ph.D. degree in Statistics from the University of California, Los Angeles, where she was working with Dr. Jingyi Jessica Li. Prior to joining UCLA, Vivian received her B.S. in Statistics from the School of Mathematics and Statistics at Huazhong University of Science and Technology.
        [ <a href="./bootstrap/doc/CV_VLi_202108.pdf" target="_blank">Vivian's CV</a> ]</p>				
        <br>
        <div style="width:15%;">
				  <p>Address:</p>
				  <p>Email:</p>
				</div>
				<div style="width:80%;">
				  <p>School of Public Health, Room 210, Rutgers University.</p>
				  <p>vivian.li@rutgers.edu</p>
				</div>
              </div> <!-- container-text -->
            </div> <!-- container-description -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->		

        <div class="container-block">
          <h1>Research Interests</h1>
          <div class="container-description">
          	<p>Our lab is interested in statistical modeling and algorithm development, as well as their applications to high-throughput sequencing data in bioinformatics and clinical research. Major research topics of our lab include:</p>
          	<ul>
          	  <li>Developing statistical and machine learning methods to understand</li>
          	  <ul>
          	  	<li>Single-cell genomic data</li>
          	  	<li>Bulk-tissue high-throughput sequencing data</li>
          	  	<li>Biomedical data</li>
          	  </ul>
          	  <li>Alternative RNA splicing and polydenylation</li> 
          	  <li>Gene regulatory networks</li> 
          	  <li>Comparative bioinformatics</li> 
          	</ul>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Research Team</h1>
          <div class="container-description">
            
            <p><b>We are looking for new team members with passion and talent!</b></p> 
            <p>We are hiring at all levels, including graduate student/research assistant/postdoc. Please contact Dr. Vivian Li if you are interested in using statistical/computational methods to answer biological questions and have a background in statistics, biostatistics, bioinformatics, computer science, or other related fields.</p>

            <h3>Current Members</h3>
            <div class="research-team-name">
              <p>Shiwei Fu</p>
              <p>Jie Sheng</p>
              <p>Dayuan Wang</p>
              <p>Andrew Cheng</p>              
            </div>
            <div class="research-team-desc">
              <p>PhD student in Biostatistics at Rutgers</p>
              <p>MS student in Data Science at UW-Madison</p>
              <p>MS student in Biostatistics at Rutgers</p>
              <p>Undergraduate student in Computer Science at Rutgers</p>
            </div>

            <h3>Alumni</h3>
            <div class="research-team-name">
              <p>Yanzeng Li</p>
              <br>
            </div>
            <div class="research-team-desc">
              <p>(12/2019 – 09/2020) MS student in Biostatistics at Rutgers <br>Now PhD student at Michigan State University</p>
            </div>

          </div> 
        </div> <!-- container-block -->

        <div id="fund-agency" class="container-block">
          <h1>Funding Agencies</h1>
          <div class="container-description">
            <div class="container-desc-col-1">
              <span><img src="./bootstrap/img/funding-agencies.png"></span>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

      </div> <!-- container-outside -->
    </script>

        <!--
        <div class="container-block">
          <h1>Honors and Awards</h1>
          <div class="container-description">
            <ul>
        <li><b>Caliburn Allocation Award of Research Computing</b>, Rutgers University, 2020</li>
        <li><b>The Most Outstanding Statistician Award</b>, UCLA, 2019</li>
        <li><b>Dissertation Year Fellowship</b> ($20,000), UCLA, 2018</li>
        <li><b>Pearl Cohen Poster Award</b>, UCLA Biomedical & Life Science Innovation Day, 2018</li>
        <li><b>Diversity Scholarship</b> ($500), UseR 2016 Conference, 2016</li>
        <li><b>Doctoral Student Travel Grant</b> ($500), UCLA, 2016</li>
        <li><b>The Most Promising Computational Statistician Award</b>, UCLA, 2015</li>
        <li><b>Distinguished Graduate of Class 2014</b>, HUST, 2014</li>
        <li><b>National Fellowship</b> (&yen;8,000), Ministry of Education ,China, 2013</li>
        <li><b>Student Award of Excellence</b> (awarded to 20 students per year), HUST, 2013</li>
        <li><b>National Fellowship</b> (&yen;8,000), Ministry of Education ,China, 2012</li>
        <li><b>National Fellowship</b> (&yen;8,000), Ministry of Education ,China, 2011</li>
        <li><b>Outstanding Student Award</b> (awarded to 3 students in each department), HUST, 2011</li>
            </ul>
          </div> 
        </div>  -->


    <!-- contact tab -->
    <script id="contact-content" type="text/x-content">
      <div class="container-outside">
        <div class="container-block">
          <h1>Rutgers University</h1>
          <div class="container-description">
           <div>
              <h3>Instructor</h3>
              <ul>
                <li>Fall 2021, BIST 0613: Biostatistics Theory I</li>
                <li>Spring 2021, BIST 0720: Advanced Biostatistical Computing</li>
                <li>Spring 2020, BIST 0725: Generalized Linear Models</li>
              </ul>
            </div>
		  </div>
        </div>

        <div class="container-block">
          <h1>University of California, Los Angeles</h1>
          <div class="container-description">

            <div>
              <h3>Graduate Student Instructor</h3>
              <ul>
	              <li>Fall 2018, STATS 495A: Teaching College Statistics</li>
              </ul>
            </div>
            <div>
              <h3>Teaching Assistant</h3>
              <ul>
                <li>Spring 2018, STATS 207: Statistical Learning with Sparsity</li>
                <li>Winter 2017, STATS 205: Hierarchical Linear Models</li>
                <li>Winter 2017, STATS 203: Large Sample Theory, Including Resampling</li>
                <li>Summer 2016, STATS 101A: Introduction to Data Analysis and Regression</li>
                <li>Spring 2016, STATS M254: Statistical Methods in Computational Biology</li>
                <li>Spring 2016, STATS 200C: Large Sample Theory</li>
                <li>Winter 2016, STATS 13: Introduction to Statistical Methods for the Health and Life Sciences</li>
                <li>Fall 2015, STATS 10: Introduction to Statistical Reasoning</li>
                <li>Summer 2015, STATS 100B: Introduction to Mathematical Statistics</li>
                <li>Spring 2015, STATS 100A: Introduction to Probability</li>
                <li>Winter 2015, STATS 10: Introduction to Statistical Reasoning</li>
                <li>Fall 2014, STATS 100A: Introduction to Probability</li>
              </ul>
            </div>			
          </div>
        </div>
      </div>
    </script>

    
    <!-- animation between tabs on navigation -->
    <script async type="text/javascript" src="./bootstrap/js/application.js"></script>
    <!-- go back to top function -->
    <script type="text/javascript" src="./bootstrap/js/backtop.js"></script>
    <!-- change gallery photo and description when click on arrows -->
    <script type="text/javascript">
      var gallery_photo_titles = new Array(
        "Welcome to Vivian Li Lab",
        "Rutgers School of Public Health",
        "Rutgers Winants Hall");
      var gallery_photo_descriptions = new Array(
        "A collection of my research and teaching activities",
        "Hornered to be an assistant professor at SPH",
        "A historic educational building located on the Queens Campus of Rutgers");
      /* set up a timer to change gallery photo in time */
      var timer = window.setInterval("changePhotoRight()", 8000);
    </script>
    <!-- extension function -->
    <script type="text/javascript" src="./bootstrap/js/extension.js"></script>

    <!-- footer shows my motto -->
    <div id="bottom-banner" class="banner">
      <div id="motto" class="wrapper">
        <span>
          <img src="./bootstrap/img/rutgers-logo-full.png">
        </span>
      </div>
    </div>
  </body>

  <!--[if lte IE 8]>
  <div id="warning">
    <p class="wrapper">*Detecting the current browser version is Microsoft Internet Explorer 8 (IE8) or below (IE7 IE6 etc.).</p>
    <p class="wrapper">**It is highly recommended to use Chrome / Firefox / IE9 to gain the best animation experience of my portfolio website.</p>
  </div>
  <![endif]-->
</html>
