<!DOCTYPE html>
<html lang="en">
  <head>
	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-172431779-1"></script>
	<script>
  		window.dataLayer = window.dataLayer || [];
  		function gtag(){dataLayer.push(arguments);}
  		gtag('js', new Date());
  		gtag('config', 'UA-172431779-1');
	</script>

    <meta charset="UTF-8">
    <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
    <meta name="description" content="Our lab is interested in statistical modeling and algorithm development, as well as their applications to high-throughput sequencing data in bioinformatics and clinical research.">
    <title>Wei Vivian Li Lab</title>
    <link rel='stylesheet' href='./bootstrap/css/style.css' type='text/css'>
    <!--[if lte IE 8]>
    <link rel='stylesheet' href='./css/style-ie.css' type='text/css'>
    <![endif]-->
    <script src="./bootstrap/js/jquery-3.3.1.min.js"></script>
  </head>

  <body>
    <div id="top-banner" class="banner"></div>
    <div id="main-body" class="wrapper">
      <div id="site-title">
        <div id="name">Wei Vivian Li
          <div id="subname">Lab of Statistical Bioinformatics</div>
        </div>
        <div id="access">
          <ul id="nav">
            <li><a id="link-home" href="#home" onclick="changeNavState(id)">HOME</a></li>
            <li><a id="link-pi" href="#pi" onclick="changeNavState(id)">PI</a></li>
            <li><a id="link-team" href="#team" onclick="changeNavState(id)">TEAM</a></li>
            <li><a id="link-research" href="#research" onclick="changeNavState(id)">RESEARCH</a></li>
            <li><a id="link-publication" href="#publication" onclick="changeNavState(id)">PUBLICATION</a></li>
            <li><a id="link-teaching" href="#teaching" onclick="changeNavState(id)">TEACHING</a></li>
          </ul>
          <select id="sel" onchange="javascript:if(this.value){location.href=this.value}">
            <option value="" selected="selected">MENU</option>
            <option value="#home">HOME</option>
            <option value="#pi">PI</option>
            <option value="#team">TEAM</option>
            <option value="#research">RESEARCH</option>
            <option value="#publication">PUBLICATION</option>
            <option value="#teaching">TEACHING</option>
          </select>
        </div>
      </div>
      <div id="content">
        <!-- will be filled by javascript -->
      </div>
    </div>
    <!-- home tab -->
    <script id="home-content" type="text/x-content">
      <div id="home-gallery">
        <div id="gallery-background" class="gallery-content">
          <span class="gallery-photo" style="opacity:1;filter:Alpha(opacity=100)">
            <img src="./bootstrap/img/gallery-photo1.jpg">
          </span>        
          <span class="gallery-photo" style="opacity:0;filter:Alpha(opacity=0)">
            <img src="./bootstrap/img/gallery-photo2.jpg">
          </span>
          <span class="gallery-photo" style="opacity:0;filter:Alpha(opacity=0)">
            <img src="./bootstrap/img/gallery-photo3.jpg">
          </span>
          <!-- two arrows switching photos -->
          <span id="arrows-left" class="gallery-arrow" onclick="window.clearInterval(timer); changePhotoLeft();timer = window.setInterval(changePhotoLeft,4000);">
            <img src="./bootstrap/img/arrows-left.png">
          </span>
          <span id="arrows-right" class="gallery-arrow" onclick="window.clearInterval(timer); changePhotoRight();timer = window.setInterval(changePhotoRight,4000);">
            <img src="./bootstrap/img/arrows-right.png">
          </span>
        </div> <!-- gallery-background -->
        <div id="gallery-text" class="gallery-content">
          <div id="gallery-description" class="container-text">
            <p id="gallery-description-showing">Welcome to Vivian Li Lab</p>
            <p id="gallery-description-hidden"></p>
          </div>
        </div> <!-- gallery-text -->
      </div> <!-- home-gallery -->
      <div id="gallery-bottom">
        <img src="./bootstrap/img/gallery-bottom.png">
      </div>

      <div class="container-outside">

        <div class="container-block">
          <h1>News</h1>
          <div id="news" class="container-description">
            <div class="container-text">
              <ul>
                <li><b>04/2024</b> New Papers! Our spVC method for the detection and interpretation of spatial gene expression variation was published at Genome Biology! The scBiG method for representation learning of single-cell gene expression data was published at NAR Genomics and Bioinformatics! </li>
                <li><b>08/2023</b> New Paper! Our manuscript, Differential variability analysis of single-cell gene expression data, was accepted by Briefings in Bioinformatics! Congratulations to Jiayi! </li>
                <li><b>10/2022</b> New Paper! Our manuscript, Benchmarking cell-type clustering methods for spatially resolved transcriptomics data, was accepted by Briefings in Bioinformatics. Congratulations to Andrew! </li>
                <li><b>06/2022</b> We are moving to University of California, Riverside this summer. We have greatly enjoyed our time at Rutgers, and we are looking forward to future endeavors at UCR! </li>
                <li><b>06/2022</b> Our scINSIGHT work has received the Best Paper Award at the RECOMB 2022 conference. Congratulations to Kun and Shiwei! </li>
                <li><b>05/2022</b> This Fall, Andrew will join Stanford Master's program in Computer Science, and Zhihan will join University of Georgia PhD program in Statistics. Congratulations to Andrew and Zhihan! </li>
              </ul>
              <details>
                <summary>Click to see all.</summary>
              <ul>
                <li><b>03/2022</b> New Paper! Our manuscript, scINSIGHT for interpreting single-cell gene expression from biologically heterogeneous data, was accepted by Genome Biology. Congratulations to Kun and Shiwei! </li>
                <li><b>03/2022</b> New Paper! Our manuscript about analyzing the homogeneity of single-cell populations was accepted by Bioinformatics! </li>
                <li><b>08/2021</b> Our R35 Maximizing Investigators' Research Award proposal was funded by the National Institute of General Medical Sciences! The grant will support our work on multi-scale analysis of single-cell transcriptomes. (<a href="https://www.newswise.com/articles/faculty-receives-national-institutes-of-health-grant-to-further-single-cell-rna-sequencing-analysis" target="_blank">Learn more</a>) </li>
                <li><b>07/2021</b> New Paper! Our manuscript, Selecting gene features for unsupervised analysis of single-cell gene expression data, was accepted by Briefings in Bioinformatics! Congratulations to Jie!</li>
                <li><b>07/2021</b> Our R21 grant proposal in collaboration with Dr. Zhiping Pang at Rutgers was funded by the National Institute of Mental Health!</li>
                <li><b>06/2021</b> New paper! Our manuscript in collaboration with Dr. Tian at the Wistar Institute, MAAPER: model-based analysis of alternative polyadenylation using 3'end-linked reads, was accepted by Genome Biology!</li>
                <li><b>05/2021</b> A visiting student in the lab, Shiwei Fu, has received the Rutgers Presidential Fellowship and will join our lab as a PhD student this fall. Congratulations to Shiwei!</li>
                <li><b>03/2021</b> New preprint! Our manuscript about analyzing alternative polyadenylation using using 3’ end-linked reads, is now available on <a href="https://www.biorxiv.org/content/10.1101/2021.03.21.436343v1" target="_blank">bioRxiv</a>.</li>
                <li><b>01/2021</b> New paper! Our manuscript, scLink: inferring sparse gene co-expression networks from single-cell expression data, was accepted by Genomics, Proteomics & Bioinformatics. Congratulations to Yanzeng!</li>
                <li><b>09/2020</b> New preprint! Our manuscript about constructing gene co-expression networks in single cells is now available on <a href="https://www.biorxiv.org/content/10.1101/2020.09.19.304956v1" target="_blank">bioRxiv</a>.</li>
                <li><b>09/2020</b> Vivian received the Busch Biomedical Grant Award.</li>
                <li><b>06/2020</b> Vivian received the BERD Method Grant from New Jersey Alliance for Clinical and Translational Science (<a href="https://njacts.rbhs.rutgers.edu/" target="_blank">NJ ACTS</a>).</li>
              </ul>
              </details>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Quick Links</h1>
          <div id="quick-link" class="container-description">
            <div class="container-text">
            <div class="container-desc-col-4">
              <a href="https://github.com/Vivianstats" target="_blank">
                <span><img src="./bootstrap/img/my-github-logo.png"></span>
              </a>
            </div>

            <div class="container-desc-col-4">
              <a href="https://twitter.com/vivianstats" target="_blank">
                <span><img src="./bootstrap/img/my-twitter-logo.png"></span>
              </a>
            </div>
            <div class="container-desc-col-4">
              <a href="https://scholar.google.com/citations?user=HzjANkQAAAAJ" target="_blank">
                <span><img src="./bootstrap/img/my-google-scholar-logo.png"></span>
              </a>
            </div>
            <div class="container-desc-col-4">
              <a href="https://www.researchgate.net/profile/Wei_Vivian_Li" target="_blank">
                <span><img src="./bootstrap/img/my-research-gate-logo.png"></span>
              </a>
            </div>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->
      </div> <!-- container-outside -->
    </script>

    <!-- research tab -->
    <script id="research-content" type="text/x-content">
      <div class="container-outside">

       <div class="container-block">
          <h1>Research Interests</h1>
          <div class="container-description">
            <div class="container-text">
              <p>Our lab conducts research at the intersection of statistics, data science, and biology. We are dedicated to developing innovative statistical and machine learning methods to address complex challenges in biological and biomedical data analysis. Our work spans foundational methodology and its application to high-throughput sequencing data in bioinformatics and clinical research. Major research topics of our lab include:</p>
              <ul>
                <li>Single-cell genomics and spatial transcriptomics</li>
                <li>Data integration and data mining</li> 
                <li>Biomedical data inference and interpretation</li> 
                <li>AI in bioinformatics</li> 
              </ul>
              Please explore our publications for the latest developments in our research.
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

	      <div class="container-block">
          <h1>Single-cell Genomics</h1>

          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-scinsight.png"></span>
            </div>
            <div class="container-text research-text">
              <h3>Integrating scRNA-seq data</h3>
              <p>The increasing number of scRNA-seq data emphasizes the need for integrative analysis to interpret similarities and differences between single-cell samples. We propose scINSIGHT to learn coordinated gene expression patterns that are common among, or specific to, different biological conditions, and identify cellular identities and processes across single-cell samples. Our results show the applicability of scINSIGHT in diverse biomedical and clinical problems. [ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02649-3" target="_blank">Paper</a> ]</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-maaper.png"></span>
            </div>
            <div class="container-text research-text">
              <h3>Analyzing Alternative Polyadenylation (APA)</h3>
              <p>MAAPER is a computational method for examining APA using 3' end-linked reads. MAAPER uses a probabilistic model to predict PASs for 3' end-linked  reads with high accuracy and sensitivity, and examines different types of APA events, including those in 3'UTRs and introns, with robust statistics. We show MAAPER's accuracy with data from both bulk and single cell RNA samples and its applicability in unpaired or paired experimental designs. [ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02429-5" target="_blank">Paper</a> ]</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-sclink.png"></span>
            </div>
            <div class="container-text research-text">
              <h3>Constructing Gene Co-expression Networks</h3>
              <p>A system-level understanding of the regulation and coordination mechanisms of gene expression is essential to studying the complexity of biological processes in health and disease. Therefore, we propose the scLink method, which uses statistical network modeling to understand the co-expression relationships among genes and construct sparse gene co-expression networks from single-cell gene expression data. [ <a href="https://doi.org/10.1016/j.gpb.2020.11.006" target="_blank">Paper</a> ]</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-scimpute.png"></span>
            </div>
            <div class="container-text research-text">
              <h3>Imputation of scRNA-seq Data</h3>
              <p>scImpute is a statistical method to accurately and robustly impute the dropouts in scRNA-seq data. scImpute automatically identifies likely dropouts, and only perform imputation on these values without introducing new biases to the rest data. scImpute is shown to identify likely dropouts, enhance the clustering of cell subpopulations, improve the accuracy of differential expression analysis, and aid the study of gene expression dynamics. [ <a href="https://www.nature.com/articles/s41467-018-03405-7" target="_blank">Paper</a> ]</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-scdesign.png"></span>
            </div>     
            <div class="container-text research-text">
              <h3>Rational scRNA-seq Experimental Design</h3>
              <p>We present a flexible and robust simulator, scDesign, the first statistical framework for researchers to quantitatively assess practical scRNA-seq experimental design in the context of differential gene expression analysis. In addition to experimental design, scDesign also assists computational method development by generating high-quality synthetic scRNA-seq datasets under customized experimental settings. 
                <p>[ <a href="https://doi.org/10.1093/bioinformatics/btz321" target="_blank">scDesign</a> ] [ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02367-2" target="_blank">scDesign2</a> ]</p>
              </p>     
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>RNA Transcript Assembly and Quantification</h1>
          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-aide.png"></span>
            </div>	    
            <div class="container-text research-text">
              <h3>Annotation-assisted Transcript Reconstruction</h3>
              <p>AIDE is a statistical method which identifies full-length mRNA isoforms from a novel perspective: using the likelihood ratio test to find novel isoforms in a stepwise manner given annotated isoforms, by prioritizing and selectively borrowing information from the annotated isoforms. AIDE is the first approach that directly controls false isoform discoveries by implementing the statistical model selection principle. [ <a href="https://genome.cshlp.org/content/29/12/2056" target="_blank">Paper</a> ]</p>      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-msiq.png"></span>
            </div>	    
            <div class="container-text research-text">
              <h3>Transcript Quantification of Multiple RNA-seq Samples</h3>
              <p>We develop a method, which we call "joint modeling of multiple RNA-seq samples for accurate isoform quantification" (MSIQ), for more robust isoform quantification by integrating multiple RNA-seq samples under a Bayesian framework. Our method aims to (1) identify a consistent group of samples with homogeneous quality and (2) improve isoform quantification accuracy by jointly modeling multiple RNA-seq samples and allowing for higher weights on the consistent group. [ <a href="https://projecteuclid.org/euclid.aoas/1520564482" target="_blank">Paper</a> ]</p>	      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Comparative Bioinformatics</h1>
          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-trom.png"></span>
            </div>	    
            <div class="container-text research-text">
              <h3>Transcriptome Overlap Measure</h3>
              <p>TROM is a testing-based method for comparing transcriptomes within or between different species, and provide a different perspective about capturing transcriptomic similarity. The TROM method focuses on identifying associated genes that capture molecular characteristics of biological samples, and subsequently comparing the biological samples by testing the overlap of their associated genes. [ <a href="https://link.springer.com/article/10.1007/s12561-016-9163-y" target="_blank">Paper</a> ]</p>	      
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-epom.png"></span>
            </div>	    
            <div class="container-text research-text">
              <h3>Epigenome Overlap Measure</h3>
              <p>We study the relationships among tissue and cell types from a new perspective: the similarity of tissue and cell types in terms of histone modification marks evaluated in relevant chromatin states. The proposed EPOM measure is capable of finding a clear correspondence map of biological samples from different tissue and cell types. The identified associated enhancers and promoters provide a comprehensive catalog to study distinct biological processes and disease variants in different tissue and cell types. [ <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2303-9" target="_blank">Paper</a> ]</p>      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Statistical Learning</h1>
          <div class="container-description research-block">
            <div class="research-photo">
              <span><img src="./bootstrap/img/research-npcs.png"></span>
            </div>
            <div class="container-text research-text">
              <h3>Asymmetric Classification</h3>
              <p>We propose an umbrella algorithm to estimate the population type I error control achieved by multiple cost-sensitive learning approaches.  Our algorithm for the first time establishes a connection between cost-sensitive learning and the Neyman-Pearson classification paradigm, which minimizes the population type II error while enforcing an upper bound on the population type I error. This algorithm can be applied in many real problems, such as disease diagnosis. [ <a href="https://arxiv.org/abs/2012.14951" target="_blank">Paper</a> ]</p>      
            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->

      </div> <!-- container-outside -->
    </script>

    <!-- publication tab -->
    <script id="publication-content" type="text/x-content">
      <div class="container-outside">
        <div class="container-block">
          <h1>Publication</h1>
          <div class="container-description">
            <div class="container-text">
              <div>
                <p>Current and previous lab members are in <b>bold</b>.</p>
                <p>Corresponding authors are <uu>underlined</uu>.</p>
              </div>
            </div>
            <div class="container-text">
<!--             	<h3>Preprints:</h3>
              <ul> 
                <li><b>Li WV</b>, Tong X, and Li JJ. (2021). Bridging cost-sensitive and Neyman-Pearson paradigms for asymmetric binary classification. <b><i>arXiv</i></b>.
                  <p>[ <a href="https://arxiv.org/abs/2012.14951" target="_blank">Link</a> ]</p></li>    

                <li>Roque W, Cuevas-Mora K, Sales D, <b>Li WV</b>, Rosas IO, and Romero F. (2020). Transcriptomics, metabolomics and lipidomics of chronically injured alveolar epithelial cells reveals similar features of IPF lung epithelium. <b><i>bioRxiv</i></b>.
                  <p>[ <a href="https://www.biorxiv.org/content/10.1101/2020.05.08.084459v1" target="_blank">Link</a> ]</p></li>
              </ul> 
-->

            	<h3>Journal Articles:</h3>
              <ul>
              <h4>2020-2024</h4>

                <li> Wan D, <b>Cheng A</b>, Wang Y, Zhong G, <b><uu>Li WV</uu></b>, <uu>Fan H</uu>. (2024). Analyzing RNA-Seq Data from Chlamydia with Super Broad Transcriptomic Activation: Challenges, Solutions, and Implications for Other Systems. <b><i>BMC Genomics</i></b>, 25, 801.
                <p>[ <a href="https://doi.org/10.1186/s12864-024-10714-3" target="_blank">Link</a> ] </p></li>

                <li> <uu>Yu S</uu> and <b><uu>Li WV</uu></b>. (2024). spVC for the detection and interpretation of spatial gene expression variation. <b><i>Genome Biology</i></b>, 25(1), 103.
                <p>[ <a href="https://doi.org/10.1186/s13059-024-03245-3" target="_blank">Link</a> ] [ <a href="https://github.com/shanyu-stat/spVC" target="_blank">Software</a> ]</p></li>

                <li> Singh M, Sahinalp C, Zeng J, <b>Li WV</b>, Kingsford C, Zhang Q, Przytycka T, Welch J, Ma J, Berger B. (2024). How has the AI boom impacted algorithmic biology? <b><i>Cell Systems</i></b>, 15(6), 483-7.
                <p>[ <a href="https://doi.org/10.1016/j.cels.2024.05.008" target="_blank">Link</a> ] </p></li>

                <li> Suntornsaratoon P, Antonio JM, Flores J, Upadhyay R, Veltri J, Bandyopadhyay S, Dadala R, Kim M, Liu Y, Balasubramanian I, Turner JR, Su X, <b>Li WV</b>, Gao N, and Ferrari RP. (2024). Lactobacillus rhamnosus GG Stimulates Dietary Tryptophan-Dependent Production of Barrier-Protecting Methylnicotinamide. <b><i>Cellular and Molecular Gastroenterology and Hepatology</i></b>, In press.
                <p>[ <a href="https://doi.org/10.1016/j.jcmgh.2024.04.003" target="_blank">Link</a> ]</p></li>

                <li> Suntornsaratoon P, <uu>Ferraris RP</uu>, Ambat J, Antonio JM, Flores J, Jones A, Su X, Gao N, and <b><uu>Li WV</uu></b>. (2024). Metabolomic and transcriptomic correlative analyses in germ-free mice link Lacticaseibacillus rhamnosus GG-associated metabolites to host intestinal fatty acid metabolism and β-oxidation. <b><i>Laboratory Investigation</i></b>, 104(4), 100330.
                <p>[ <a href="https://doi.org/10.1016/j.labinv.2024.100330" target="_blank">Link</a> ]</p></li>

                <li> Li T, <b>Qian K</b>, Wang X, <b><uu>Li WV</uu></b>, and <uu>Li H</uu>. (2024). scBiG for representation learning of single-cell gene expression data based on bipartite graph embedding. <b><i>NAR Genomics and Bioinformatics</i></b>, 6(1), lqae004.
                <p>[ <a href="https://doi.org/10.1093/nargab/lqae004" target="_blank">Link</a> ] [ <a href="https://github.com/sldyns/scBiG" target="_blank">Software</a> ]</p></li>

                <li> Wurihan W, Wang Y, Yeung S, Zou Y, Lai Z, Fondell JD, <b>Li WV</b>, Zhong G, and Fan H. (2024). Expression activation of over 70% of Chlamydia trachomatis genes during the first hour of infection. <b><i>Infection and Immunity</i></b>, online ahead of print.
                <p>[ <a href="https://doi.org/10.1128/iai.00539-23" target="_blank">Link</a> ]</p></li>

                <li> Lu B, Wang Y, Wurihan W, <b>Cheng A</b>, Yeung S, Fondell JD, Lai Z, Wan D, Wu X, <b>Li WV</b>, and Fan H. (2024). Requirement of GrgA for Chlamydia infectious progeny production, optimal growth, and efficient plasmid maintenance. <b><i>Mbio</i></b>, 15(1), e02036-23.
                <p>[ <a href="https://doi.org/10.1128/mbio.02036-23" target="_blank">Link</a> ]</p></li>

                <li> Ren H, Chen X, Wang J, Chen Y, Hafiz A, Xiao Q, <b>Fu S</b>, Madireddy A, <b>Li WV</b>, Shi X, and Cao J. (2023). Temporal and structural patterns of hepatitis B virus integrations in hepatocellular carcinoma. <b><i>Journal of Medical Virology</i></b>, 95(10), e29187.
                <p>[ <a href="https://doi.org/10.1002/jmv.29187" target="_blank">Link</a> ]</p></li>

                <li> Balasubramanian I, Bandyopadhyay S, Flores J, Bianchi-Smak Jared, Lin X, Liu H, Sun S, Golovchenko N, Liu Y, Wang D, Patel R, Joseph I, Suntornsaratoon P, Vargas J, Green P, Bhagat G, Lagana S, Ying W, Zhang Y, <b>Wang Z</b>, <b>Li WV</b>, Singh S, Zhou Z, Kollias G, Farr L, Moonah S, Yu S, Wei Z, Bonder E, Zhang L, Kiela P, Edelblum K, Ferraris R, Liu T, and Gao N. (2023). Infection and inflammation stimulate expansion of a CD74+ Paneth cell subset to regulate disease progression. <b><i>The EMBO Journal</i></b>, e113975.
                  <p>[ <a href="https://doi.org/10.15252/embj.2023113975" target="_blank">Link</a> ]</p></li>

                <li> He X, <b>Qian K</b>, Wang Z, Zeng S, <uu>Li H</uu>, and <b><uu>Li WV</uu></b>. (2023). scAce: an adaptive embedding and clustering method for single-cell gene expression data. <b><i>Bioinformatics</i></b>, btad546.
                  <p>[ <a href="https://doi.org/10.1093/bioinformatics/btad546" target="_blank">Link</a> ] [ <a href="https://github.com/sldyns/scAce" target="_blank">Software</a> ]</p></li>

                <li><b>Liu J</b>, <uu>Kreimer A</uu>, and <b><uu>Li WV</uu></b>. (2023). Differential variability analysis of single-cell gene expression data. <b><i>Briefings in Bioinformatics</i></b>, bbad294.
                  <p>[ <a href="https://doi.org/10.1093/bib/bbad294" target="_blank">Link</a> ]</p></li>

                <li><b>Cheng A</b>, Hu G, and <b><uu>Li WV</uu></b>. (2022). Benchmarking cell-type clustering methods for spatially resolved transcriptomics data. <b><i> Briefings in Bioinformatics</i></b>, bbac475.
                  <p>[ <a href="https://doi.org/10.1093/bib/bbac475" target="_blank">Link</a> ]</p></li>

                <li> Berger B, Tian D, <b>Li WV</b>, El-Kebir M, Tomescu AI, Singh R, Beerenwinkel N, Li Y, Boucher C, and Bar-Joseph Z. (2022). What are the keys to succeeding as a computational biologist in today’s research climate? <b><i>Cell Systems</i></b>, 13(10), 781-785.
                  <p>[ <a href="https://doi.org/10.1016/j.cels.2022.09.005" target="_blank">Link</a> ] </p></li>  

                <li> <b>Qian K</b>, <b>Fu S</b>, Li H, and <b><uu>Li WV</uu></b>. (2022). The scINSIGHT package for integrating single-cell RNA-seq data from different biological conditions. <b><i>Journal of Computational Biology</i></b>, Online Ahead of Print.
                  <p>[ <a href="https://doi.org/10.1089/cmb.2022.0244" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scINSIGHT" target="_blank">Software</a> ] [ <a href="https://recomb2022.net/accepted-papers/" target="_blank">RECOMB 2022 Best Paper Award</a> ]</p></li> 

                <li> <b>Qian K</b>, <b>Fu S</b>, Li H, and <b><uu>Li WV</uu></b>. (2022). scINSIGHT for interpreting single-cell gene expression from biologically heterogeneous data. <b><i>Genome Biology</i></b>, 23, 82.
                  <p>[ <a href="https://doi.org/10.1186/s13059-022-02649-3" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scINSIGHT" target="_blank">Software</a> ]</p></li>  

                <li><b><uu>Li WV</uu></b>. (2022). Phitest for analyzing the homogeneity of single-cell populations. <b><i>Bioinformatics</i></b>, btac130.
                  <p>[ <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac130/6541626?guestAccessKey=4cbe9bd6-f46c-4780-b7c1-4c17af176917" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/PhitestR" target="_blank">Software</a> ]</p></li> 

                <li>Sun T, Song D, <b>Li WV</b>, and Li JJ. (2022). Simulating single-cell gene expression count data with preserved gene correlations by scDesign2. <b><i>Journal of Computational Biology</i></b>, 29(1), 23-26.
                  <p>[ <a href="https://doi.org/10.1089/cmb.2021.0440" target="_blank">Link</a> ]</p></li> 

                <li><b><uu>Li WV</uu></b> and <b>Li Y</b>. (2021). scLink: inferring sparse gene co-expression networks from single-cell expression data. <b><i>Genomics, Proteomics & Bioinformatics</i></b>, 19(3), 475-492.
                  <p>[ <a href="https://doi.org/10.1016/j.gpb.2020.11.006" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scLink" target="_blank">Software</a> ] [ <a href="https://rutgersbiostat.shinyapps.io/sclink/" target="_blank">Web App</a> ] [ <a href="https://www.recomb2021.org/accepted-papers" target="_blank">RECOMB 2021</a> ]</p></li>

                <li>Colon A, Hirday R, Patel A, Poddar A, Tuberty-Vaughan E, Fu T, Ai X, <b>Li WV</b>, and Li C. (2021). A computational pipeline for functional gene discovery. <b><i>Scientific Reports</i></b>, 11, 23522.
                  <p>[ <a href="https://doi.org/10.1038/s41598-021-03041-0" target="_blank">Link</a> ]</p></li> 
                <li><b><uu>Li WV</uu></b>, Zheng D, Wang R, and <uu>Tian B</uu>. (2021). MAAPER: model-based analysis of alternative polyadenylation using 3'end-linked reads. <b><i>Genome Biology</i></b>, 22, 222.
                  <p>[ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02429-5" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/MAAPER" target="_blank">Software</a> ]</p></li>

                <li><b>Sheng J</b> and <b><uu>Li WV</uu></b>. (2021). Selecting gene features for unsupervised analysis of single-cell gene expression data. <b><i> Briefings in Bioinformatics</i></b>, bbab295.
                  <p>[ <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab295/6341636?guestAccessKey=2f8cc725-e1bc-4ff1-905b-504931532ee4" target="_blank">Link</a> ]</p></li>

                <li>Zhang XS, Yin YS, Wang J, Battaglia T, Krautkramer K, <b>Li WV</b>, Li J, Brown M, Zhang M, Badri MH, Armstrong AJ, Strauch CM, Wang Z, Nemet I, Altomare N, Devlin JC, He L, Morton JT, Chalk JA, Needles K, Liao V, Mount J, Li H, Ruggles KV, Bonneau RA, Dominguez-Bello MG, Bäckhed F, Hazen SL, Blaser MJ. (2021). Maternal cecal microbiota transfer rescues early-life antibiotic-induced enhancement of type 1 diabetes in mice. <b><i>Cell Host & Microbe</i></b>, 29(8):1249-1265.
                  <p>[ <a href="https://doi.org/10.1016/j.chom.2021.06.014" target="_blank">Link</a> ]</p></li>

                <li>Sun T, Song D, <b><uu>Li WV</uu></b>, and <uu>Li JJ</uu>. (2021).  scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured. <b><i>Genome Biology</i></b>, 22(1):1-37.
                  <p>[ <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02367-2" target="_blank">Link</a> ] [ <a href="https://www.recomb2021.org/accepted-papers" target="_blank">RECOMB 2021</a> ] </p></li>

                <li>Jiang R, <b>Li WV</b>, and Li JJ. (2021). mbImpute: an accurate and robust imputation method for microbiome data. <b><i>Genome Biology</i></b>, 22:192.
                  <p>[ <a href="https://doi.org/10.1186/s13059-021-02400-4" target="_blank">Link</a> ]</p></li>

                <li>Zheng Y, Chen Z, Han L, Han Y, Zou X, Zhou B, Hu R, Hao J, Bai S, Xiao H, <b>Li WV</b>, Bueker A, Ma Y, Xie G, Yang J, Chen S, Li H, Cao J, and Shen L. (2020). Immune suppressive landscape in the human esophageal squamous cell carcinoma microenvironment. <b><i>Nature Communications</i></b>, 11(1):1-17.  
                  <p>[ <a href="https://www.nature.com/articles/s41467-020-20019-0" target="_blank">Link</a> ]</p></li>  

                <li>Wang J, Deng F, Zeng F, Shanahan AJ, <b>Li WV</b>, and Zhang L. (2020). Predicting long-term multicategory cause of death in patients with prostate cancer: random forest versus multinomial model. <b><i>American Journal of Cancer Research</i></b>, 10(5):1344.
                  <p>[ <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7269775/" target="_blank">Link</a> ]</p></li>

                <li>Xu J, Cheng Y, Yuan X, <b>Li WV</b>, and Zhang L. (2020). Trends and prediction in daily incidence of novel coronavirus infection in China, Hubei Province and Wuhan City: an application of Farr's law. <b><i>American Journal of Translational Research</i></b>, 12(4):1355.
                  <p>[ <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7191162/" target="_blank">Link</a> ]</p></li>  

              <h4>2016-2019</h4>

                <li><b>Li WV</b>*, Li S*, Tong X, Deng L, Shi H, and Li JJ. (2019). AIDE: annotation-assisted isoform discovery and abundance estimation from RNA-seq data. <b><i>Genome Research</i></b>, 29:2056-2072.
                  <p>[ <a href="https://genome.cshlp.org/content/29/12/2056" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/AIDE" target="_blank">Software</a> ] [ <a href="https://genome.cshlp.org/content/29/12.cover-expansion" target="_blank">Cover Story</a> ] [ <a href="https://www.aspph.org/rutgers-study-achieves-highest-precision-and-lowest-error-rates-in-mrna-discoveries-using-a-robust-bioinformatics-tool/" target="_blank">ASPPH News</a> ]</p></li>

                <li><b>Li WV</b> and Li JJ. (2019). A statistical simulator scDesign for rational scRNA-seq experimental design. <b><i>Bioinformatics</i></b>, 35(14):i41-i50.
                  <p>[ <a href="https://academic.oup.com/bioinformatics/article/35/14/i41/5529133" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scDesign" target="_blank">Software</a> ]</p></li>

                <li>Ge X, Zhang H, Xie L, <b>Li WV</b>, Kwon SB, and Li JJ. (2019). EpiAlign: an alignment-based bioinformatic tool for comparing chromatin state sequences. <b><i>Nucleic Acids Research</i></b>, gkz287. 
                  <p>[ <a href="https://doi.org/10.1093/nar/gkz287" target="_blank">Link</a> ] [ <a href="http://shiny.stat.ucla.edu:3838/EpiAlign/" target="_blank">Website</a> ] </p></li>

                <li><b>Li WV</b> and Li JJ. (2018). An accurate and robust imputation method scImpute for single-cell RNA-seq data. <b><i>Nature Communications</i></b>, 9:997.
  	             <p>[ <a href="https://www.nature.com/articles/s41467-018-03405-7" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/scImpute" target="_blank">Software</a> ] [ <a href="http://newsroom.ucla.edu/releases/ucla-scientists-merge-statistics-biology-to-produce-important-new-gene-computational-tool" target="_blank">UCLA News</a> ]</p></li>

                <li><b>Li WV</b>*, Zhao A, Zhang S, and Li JJ*. (2018). MSIQ: joint modeling of multiple RNA-seq samples for accurate isoform quantification. <b><i>Annals of Applied Statistics</i></b>, 12(1):510-539. 
  	               <p>[ <a href="https://projecteuclid.org/euclid.aoas/1520564482" target="_blank">Link</a> ] [ <a href="https://github.com/Vivianstats/MSIQ" target="_blank">Software</a> ]</p></li>

  	            <li><b>Li WV</b> and Li JJ. (2018). Modeling and analysis of RNA-seq data: a review from a statistical perspective. <b><i>Quantitative Biology</i></b>, 6(3):195-209. 
  	              <p>[ <a href="https://link.springer.com/article/10.1007/s40484-018-0144-7" target="_blank">Link</a> ]</p></li>

                <li><b>Li WV</b>, Chen Y, and Li JJ. (2017). TROM: a testing-based method for finding transcriptomic similarity of biological samples. <b><i>Statistics in Biosciences</i></b>, 9(1):105-136.
  	             <p>[ <a href="https://link.springer.com/article/10.1007/s12561-016-9163-y" target="_blank">Link</a> ] [ <a href="https://cran.r-project.org/web/packages/TROM/index.html" target="_blank">Software</a> ]</p></li>

                <li><b>Li WV</b>, Razaee ZS, and Li JJ. (2016). Epigenome overlap measure (EPOM) for comparing tissue/cell types based on chromatin states. <b><i>BMC Genomics</i></b>, 17(Supp 1):10. 
  	             <p>[ <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2303-9" target="_blank">Link</a> ] [ <a href="https://github.com/ruochenj/EPOM_R" target="_blank">Software</a> ]</p></li>
              </ul>

              <h3>Conference Articles:</h3>
              <ul>
                <li> <b>Qian K</b>, <b>Fu S</b>, Li H, and <b><uu>Li WV</uu></b>. (2022). A Novel Matrix Factorization Model for Interpreting Single-Cell Gene Expression from Biologically Heterogeneous Data. In <b><i>International Conference on Research in Computational Molecular Biology</i></b>. Springer, Cham. </li>              
              </ul>

              <h3>Dissertation:</h3>
              <ul>
                <li><b>Li, WV</b> (2019). Statistical methods for bulk and single-cell RNA sequencing data. PhD dissertation, <b><i>UCLA</i></b>.
  	            <p>[ <a href="https://escholarship.org/uc/item/50z7659w" target="_blank">Link</a> ]</p></li>	             
              </ul>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Software</h1>
          <div class="container-description">
            <div class="container-text">
              <h3>Single-cell Transcriptomics</h3>
              <div class="software-text">
                <ul>
                  <li><p><a href="https://github.com/Vivianstats/scINSIGHT" target="_blank">scINSIGHT</a>: Integrating single-cell gene expression data from multiple biological conditions</p></li>
        	        <li><p><a href="https://github.com/Vivianstats/scImpute" target="_blank">sclmpute</a>: Accurate and robust imputation of scRNA-seq data</p></li>
        	        <li><p><a href="https://github.com/Vivianstats/scDesign" target="_blank">scDesign</a>: A statistical simulator for rational single-cell experimental design</p></li>
                  <li><p><a href="https://github.com/Vivianstats/scLink" target="_blank">scLink</a>: Inferring sparse gene co-expression networks in single cells</p></li>
                  <li><p><a href="https://github.com/Vivianstats/MAAPER" target="_blank">MAAPER</a>: Analyzing alternative polyadenylation using 3' end-linked reads</p></li>
                  <li><p><a href="https://github.com/Vivianstats/Phitest" target="_blank">Phitest</a>: Analyzing the homogeneity of single-cell populations</p></li>

                </ul>
              </div>                          
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description">
            <div class="container-text">
      	      <h3>Bulk Tissue Transcriptomics</h3>
              <div class="software-text">
        	      <ul>
        	        <li><p><a href="https://github.com/Vivianstats/AIDE" target="_blank">AIDE</a>: Annotation-assisted transcript reconstruction and quantification</p></li>
                  <li><p><a href="https://github.com/Vivianstats/MSIQ" target="_blank">MSIQ</a>: Quantification of RNA transcripts from multiple RNA-seq samples</p></li>
                  <li><p><a href="https://github.com/Vivianstats/MAAPER" target="_blank">MAAPER</a>: Analyzing alternative polyadenylation using 3' end-linked reads</p></li>          
        	      </ul>
              </div>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description">
            <div class="container-text">
              <h3>Comparative Bioinformatics</h3>
              <div class="software-text">
                <ul>
                  <li><p><a href="https://github.com/Vivianstats/TROM" target="_blank">TROM</a>: Comparing transcriptomes from the same or different species</p></li>
                  <li><p><a href="https://github.com/ruochenj/EPOM_R" target="_blank">EPOM</a>: Comparing epigenomes of biological samples</p></li>
                </ul>
              </div>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

        </div> <!-- container-block -->
      </div>  <!-- container-outside -->
    </script>

    <!-- PI tab -->
    <script id="pi-content" type="text/x-content">
      <div class="container-outside">
        <div class="container-block">
          <h1>About PI</h1>
          <div class="container-description">
            <div id="about-me-photo">
              <span><img src="./bootstrap/img/about-me.png"></span>
            </div>
            <div id="about-me-text" class="container-text">
              <p>Wei (Vivian) Li is an Assistant Professor in the <a href="https://statistics.ucr.edu/" target="_blank">Department of Statistics</a> at University of California, Riverside. She is also a faculty member of the UCR graduate program in <a href="https://ggb.ucr.edu/" target="_blank">Genetics, Genomics & Bioinformatics</a>. Previously, she was an Assistant Professor at Rutgers Department of Biostatistics and Epidemiology. Vivian obtained her Ph.D. degree in Statistics from the University of California, Los Angeles, where she was working with Dr. Jingyi Jessica Li. Prior to joining UCLA, Vivian received her B.S. in Statistics from the School of Mathematics and Statistics at Huazhong University of Science and Technology. </p>	
              <p> [ <a href="./bootstrap/doc/CV_WeiLi_202110.pdf" target="_blank">Vivian's CV</a> ]</p>

              <div class="about-me-text-info">
                <p style="font-weight:bold;width:15%;">Address:</p>
                <p style="width:80%;">MRB 4133, UCR.</p>
              </div>
              <div class="about-me-text-info">
                <p style="font-weight:bold;width:15%;">Email:</p>
                <p style="width:80%;">weil@ucr.edu</p>
              </div>

            </div> <!-- container-text -->
          </div> <!-- container-description -->
        </div> <!-- container-block -->		

        <div class="container-block">
          <h1>Research Interests</h1>
          <div class="container-description">
            <div class="container-text">
            	<p>Our lab conducts research at the intersection of statistics, data science, and biology. We are dedicated to developing innovative statistical and machine learning methods to address complex challenges in biological and biomedical data analysis. Our work spans foundational methodology and its application to high-throughput sequencing data in bioinformatics and clinical research. Major research topics of our lab include:</p>
            	<ul>
            	  <li>Single-cell genomics and spatial transcriptomics</li>
            	  <li>Data integration and data mining</li> 
            	  <li>Biomedical data inference and interpretation</li> 
            	  <li>AI in bioinformatics</li> 
            	</ul>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div id="fund-agency" class="container-block">
          <h1>Funding Agencies</h1>
          <div class="container-description">
            <div class="container-desc-col-2">
              <span><img src="./bootstrap/img/funding-agency-nih.png"></span>
            </div>
            <div class="container-desc-col-2">
              <span><img src="./bootstrap/img/funding-agency-nj.png"></span>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

      </div> <!-- container-outside -->
    </script>

    <!-- team tab -->
    <script id="team-content" type="text/x-content">
      <div class="container-outside">

        <div class="container-block">
          <h1>Current Members</h1>
          <div class="container-description">
            <div class="container-text">
              <p><b>We are looking for new team members with passion and talent!</b></p> 
              <p><b>We aim to foster an inclusive team culture that values diverse backgrounds and perspectives.</b></p> 
              <p>We are hiring at all levels, including graduate student/research assistant/postdoc. We also welcome intern students for research training, both remotely and on-site. Please contact Dr. Vivian Li if you are interested in using statistical and/or computational methods to answer biological questions and have a background in statistics, biostatistics, bioinformatics, computer science, or other related fields.</p>
            </div>
          </div>

          <div class="container-description">
            <div class="team-member-photo">
              <span><img src="./bootstrap/img/team-marui.png"></span>
            </div>
            <div class="container-text team-member-text">
              <p><b>Rui Ma</b></p> 
              <p>Postdoctoral Researcher (co-advised by Prof. Wenxiu Ma)</p>
              <p>Rui is a postdoc in the Department of Statistics at UC Riverside. She received her PhD in Applied Statistics and her MS in Computer Science from UC Riverside. Rui is interested in single-cell omic analysis, with a focus on single-cell Hi-C.</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description">
            <div class="team-member-photo">
              <span><img src="./bootstrap/img/team-fushiwei.png"></span>
            </div>
            <div class="container-text team-member-text">
              <p><b>Shiwei Fu</b></p> 
              <p>PhD student</p>
              <p>Shiwei is a PhD student in the Department of Statistics. Before joining UCR, he completed an MS degree in Statistics at University of Illinois at Urbana–Champaign. Shiwei is interested in single-cell RNA-seq analysis and other emerging transcriptomic sequencing technologies.</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description">
            <div class="team-member-photo">
              <span><img src="./bootstrap/img/team-muyudi.png"></span>
            </div>
            <div class="container-text team-member-text">
              <p><b>Yudi Mu</b></p> 
              <p>PhD student (co-advised by Prof. Wenxiu Ma)</p>
              <p>Yudi is a PhD Student in the Department of Statistics. Previously, she obtained an MS degree at University of Wisconsin–Madison. She is mainly interested in bioinformatics.</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description">
            <div class="team-member-photo">
              <span><img src="./bootstrap/img/team-yangtianjian.png"></span>
            </div>
            <div class="container-text team-member-text">
              <p><b>Tianjian (Kiano) Yang</b></p> 
              <p>PhD student</p>
              <p>Kiano is a PhD student in the Department of Statistics. Before joining UCR, he completed an MS degree at UC Los Angeles. His interests broadly lie in bioinformatics, statistical genetics, high-dimensional statistics, and machine learning.</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->

          <div class="container-description">
            <div class="team-member-photo">
              <span><img src="./bootstrap/img/team-tranbrian.png"></span>
            </div>
            <div class="container-text team-member-text">
              <p><b>Brian Tran</b></p> 
              <p>PhD student</p>
              <p>Brian is a PhD student in the Department of Statistics and received his MS in Statistics at UCR. His research interests include biostatistics, bioinformatics, and applications of statistics in biological problems.</p>
            </div> <!-- container-text -->
          </div> <!-- container-description -->


        </div> <!-- container-block -->


        <div class="container-block">
          <h1>Alumni</h1>
          <div class="container-description">
            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Anika Singh</b></p>
              </div>
              <div class="team-member-text">
                <p>(07/2023 – 08/2023) Undergraduate student in Mathematics at Pasadena City College (NSF REU Program)</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Jiayi Liu</b></p>
              </div>
              <div class="team-member-text">
                <p>(09/2021 – 08/2023) Rotation PhD student in Molecular Biosciences at Rutgers</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Andrew Cheng</b></p>
              </div>
              <div class="team-member-text">
                <p>(09/2020 – 11/2022) Undergraduate student in Computer Science at Rutgers</p>
                <p>&rarr; MS student at Stanford University</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Lloyd Gilfillian</b></p>
              </div>
              <div class="team-member-text">
                <p>(08/2022 – 09/2022) Rotation student (UCR JumpStart program)</p>
                <p>&rarr; PhD student at UCR</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Zhihan Wang</b></p>
              </div>
              <div class="team-member-text">
                <p>(09/2021 – 05/2022) MS student in Statistics at Rutgers</p>
                <p>&rarr; PhD student at University of Georgia</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Jie Sheng</b></p>
              </div>
              <div class="team-member-text">
                <p>(11/2020 – 07/2021) MS student in Data Science at University of Wisconsin–Madison</p>
                <p>&rarr; Researcher at University of Wisconsin–Madison</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Dayuan Wang</b></p>
              </div>
              <div class="team-member-text">
                <p>(08/2020 – 12/2020) MS student in Biostatistics at Rutgers</p>
                <p>&rarr; PhD student at University of South Carolina</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Kun Qian</b></p>
              </div>
              <div class="team-member-text">
                <p>(11/2019 – 12/2021) MS student in Statistics at CUG</p>
                <p>&rarr; PhD student at Peking University</p>
              </div>
            </div>

            <div class="container-text team-alumni-block">
              <div class="team-member-photo">
                <p><b>Yanzeng Li</b></p>
              </div>
              <div class="team-member-text">
                <p>(12/2019 – 09/2020) MS student in Biostatistics at Rutgers</p>
                <p>&rarr; PhD student at Michigan State University</p>
              </div>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->
      </div> <!-- container-outside -->
    </script>

    <!-- teaching tab -->
    <script id="teaching-content" type="text/x-content">
      <div class="container-outside">
        <div class="container-block">
          <h1>University of California, Riverside</h1>
          <div class="container-description">
           <div class="container-text">
              <h3>Instructor</h3>
              <ul>
                <li>Spring 2024, STAT 140: Nonparametric Statistical Techniques</li>
                <li>Winter 2024, STAT 208: Statistical Data Mining Methods</li>
                <li>Fall 2023, STAT 218: Survival Analysis</li>
                <li>Spring 2023, STAT 140: Nonparametric Statistical Techniques</li>
                <li>Winter 2023, STAT 208: Statistical Data Mining Methods</li>
                <li>Fall 2022, STAT 218: Survival Analysis</li>
              </ul>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>Rutgers University</h1>
          <div class="container-description">
           <div class="container-text">
              <h3>Instructor</h3>
              <ul>
                <li>Spring 2022, BIST 0725: Generalized Linear Models</li>
                <li>Fall 2021, BIST 0613: Biostatistics Theory I</li>
                <li>Spring 2021, BIST 0720: Advanced Biostatistical Computing</li>
                <li>Spring 2020, BIST 0725: Generalized Linear Models</li>
              </ul>
            </div>
          </div> <!-- container-description -->
        </div> <!-- container-block -->

        <div class="container-block">
          <h1>University of California, Los Angeles</h1>
          <div class="container-description">
            <div class="container-text">
              <h3>Graduate Student Instructor</h3>
              <ul>
	              <li>Fall 2018, STATS 495A: Teaching College Statistics</li>
              </ul>
            </div>
            <div class="container-text">
              <h3>Teaching Assistant</h3>
              <ul>
                <li>Spring 2018, STATS 207: Statistical Learning with Sparsity</li>
                <li>Winter 2017, STATS 205: Hierarchical Linear Models</li>
                <li>Winter 2017, STATS 203: Large Sample Theory, Including Resampling</li>
                <li>Summer 2016, STATS 101A: Introduction to Data Analysis and Regression</li>
                <li>Spring 2016, STATS M254: Statistical Methods in Computational Biology</li>
                <li>Spring 2016, STATS 200C: Large Sample Theory</li>
                <li>Winter 2016, STATS 13: Introduction to Statistical Methods for the Health and Life Sciences</li>
                <li>Fall 2015, STATS 10: Introduction to Statistical Reasoning</li>
                <li>Summer 2015, STATS 100B: Introduction to Mathematical Statistics</li>
                <li>Spring 2015, STATS 100A: Introduction to Probability</li>
                <li>Winter 2015, STATS 10: Introduction to Statistical Reasoning</li>
                <li>Fall 2014, STATS 100A: Introduction to Probability</li>
              </ul>
            </div>			
          </div> <!-- container-description -->
        </div> <!-- container-block -->
      </div> <!-- container-outside -->
    </script>

    <!-- animation between tabs on navigation -->
    <script async type="text/javascript" src="./bootstrap/js/application.js"></script>
    <!-- go back to top function -->
    <script type="text/javascript" src="./bootstrap/js/backtop.js"></script>
    <!-- change gallery photo and description when click on arrows -->
    <script type="text/javascript">
      var gallery_photo_descriptions = new Array(
        "Welcome to Vivian Li Lab",
        "Our Selected Statistical Tools",
        "UC Riverside Campus");
      var gallery_photo_titles = gallery_photo_descriptions; /* legacy */
      /* set up a timer to change gallery photo in time */
      var timer = window.setInterval("changePhotoRight()", 8000);
    </script>
    <!-- extension function -->
    <script type="text/javascript" src="./bootstrap/js/extension.js"></script>

    <!-- footer shows my motto -->
    <div id="bottom-banner" class="banner">
      <div id="motto" class="wrapper">
        <span>
          <img src="./bootstrap/img/footer-logo-full.png">
        </span>
      </div>
    </div>
  </body>

  <!--[if lte IE 8]>
  <div id="warning">
    <p class="wrapper">*Detecting the current browser version is Microsoft Internet Explorer 8 (IE8) or below (IE7 IE6 etc.).</p>
    <p class="wrapper">**It is highly recommended to use Chrome / Firefox / IE9 to gain the best experience of this website.</p>
  </div>
  <![endif]-->
</html>
